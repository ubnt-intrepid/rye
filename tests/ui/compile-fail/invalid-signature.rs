fn main() {}

#[rye::test]
fn has_input(x: u32) {}

#[rye::test]
fn has_generics<T>() {}
