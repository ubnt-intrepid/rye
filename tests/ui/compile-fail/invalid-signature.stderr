error: test functions cannot accept arguments
 --> $DIR/invalid-signature.rs:4:14
  |
4 | fn has_input(x: u32) {}
  |              ^^^^^^

error: test functions cannot take generic parameters
 --> $DIR/invalid-signature.rs:7:16
  |
7 | fn has_generics<T>() {}
  |                ^^^
